### ISSUE TITLE
Mention running autogen.sh in README?

### ISSUE THREAD

**Issue by jibnew** [4ti2/4ti2#35]
This is doubtlessly a minor issue but an inexperienced user might be at a loss when said to run `./configure` which is not there.
Actually, although I compiled 4ti2 successfully and it works, I am not sure whether the canonical way is to first of all run `autogen.sh` and then follow the steps in README. Is it so?

**Comment by mkoeppe** [comment]
Yes. For historical reasons, this information is in the file `README.cvs`.
